tools/run_tests/sanity/check_submodules.sh

```diff
@@ -37,7 +37,7 @@ third_party/opencensus-proto 4aa53e15cbf1a47bc9087e6cfdca214c1eea4e89
 third_party/opentelemetry 60fa8754d890b5c55949a8c68dcfd7ab5c2395df
 third_party/opentelemetry-cpp 4bd64c9a336fd438d6c4c9dad2e6b61b0585311f
 third_party/protobuf 10ef3f77683f77fb3c059bf47725c27b3ff41e63
-third_party/protoc-gen-validate fab737efbb4b4d03e7c771393708f75594b121e4
+third_party/protoc-gen-validate 32c2415389a3538082507ae537e7edd9578c64ed
```

Comment:
This is remove above, should it be gone from here as well?

Reply:
It wasn't removed, it was upgraded to a new version.